vuepress/docs/v7.0/tutorials/create/mfe/react.md

```diff
@@ -172,20 +170,19 @@ Now add a widget for your MFE.
 
 ### View the Widget
 
-View the React micro frontend in action on a page.
+Place the React micro frontend onto a page to see it in action.
 
-1. In the `Entando App Builder` go back to `Pages` → `Management` 
+1. In the `Entando App Builder`, go to `Pages` → `Management` 
 
-2. Choose an existing page (or create a new one (LINK TODO)) and click `Actions`→ `Design`. 
+2. Choose an existing page (or [create a new one](../../compose/page-management.md#create-a-page)) and select `Design` from its Actions
```

Comment:
debating whether the Actions should be backticked

Reply:
i opted not to because it's not a label or entity